cinema_service/settings.py

```diff
@@ -134,7 +140,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
MEDIA_ROOT is set to a static path '/files/media'. It's better to use a dynamic path like os.path.join(BASE_DIR, 'media') to ensure compatibility across different environments.